test/test_cases/run_1d_ekman_column.jl

```diff
@@ -81,8 +81,11 @@ function run_1d_ekman_column(
         # Generate simple simulation data for test
         simulation = Simulation(model, stepper, dt = dt, tspan = (0.0, dt))
         @unpack ρ, uv, w, ρθ = init_1d_ekman_column(FT, params)
-        set!(simulation, :base, ρ = ρ, uh = uv, w = w)
-        set!(simulation, :thermodynamics, ρθ = ρθ)
+        Y = simulation.integrator.u
+        set!(Y.base.ρ, ρ)
```

Comment:
Oh I see the problem... Here `@unpack ρ, uv, w, ρθ = init_1d_ekman_column(FT, params)` gives `ρ, uv, w, ρθ` as functions not the fields.